sunpy/tests/helpers.py

```diff
@@ -63,48 +63,12 @@
 
 skip_wcsaxes = pytest.mark.skipif(SKIP_WCSAXES, reason="wcsaxes is not available")
 
-
 @pytest.fixture
 def warnings_as_errors(request):
     warnings.simplefilter('error')
 
     request.addfinalizer(lambda *args: warnings.resetwarnings())
 
-
-def assert_quantity_allclose(actual, desired, rtol=1.e-7, atol=0, err_msg='', verbose=True):
```

Comment:
I feel as though removing this in a bug fix release is not ok. It might be a helper, but it is still technically public API.

PUT IT BACK.